winpr/libwinpr/synch/wait.c

```diff
@@ -109,7 +109,7 @@ static long long ts_difftime(const struct timespec *o,
 #if !defined(HAVE_PTHREAD_GNU_EXT)
 #include <pthread.h>
 
-#if defined(__FreeBSD__) || defined(sun)
+#if defined(__FreeBSD__) || defined(sun) || defined(__OpenBSD__)
```

Comment:
So there is actually an implementation available on OpenBSD? Then this custom implementation should not be used at all, it is meant for systems lacking one.